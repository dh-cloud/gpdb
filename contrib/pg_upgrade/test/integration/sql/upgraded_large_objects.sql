-- there should be no entry in large object table
SELECT count(*) FROM pg_largeobject;
