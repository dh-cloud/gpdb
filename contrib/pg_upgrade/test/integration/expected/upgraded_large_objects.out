-- there should be no entry in large object table
SELECT count(*) FROM pg_largeobject;
 count 
-------
 0     
(1 row)
